db/migrate/20241031112333_add_kind_to_talk.rb

```diff
@@ -0,0 +1,9 @@
+class AddKindToTalk < ActiveRecord::Migration[8.0]
+  def change
+    add_column :talks, :kind, :string, null: true
+
+    Talk.update_all kind: :standard
+
+    change_column_null :talks, :kind, false
```

Comment:
Would it make sense to just set the default value to `:standard`?

```suggestion
    add_column :talks, :kind, :string, default: "standard", null: false
```

Reply:
I started by adding a default value at the database level, but ran into some issues with tests. I'll check if adding [these conditions in the callback](https://github.com/adrienpoly/rubyvideo/pull/334#discussion_r1827708596) solves the problem.